docs/lib/project-setup/fragments/ios/project-setup/51_provisionBackend.md

```diff
@@ -0,0 +1,13 @@
+To add these configuration files to your project, **open finder within your project** and **drag both `amplifyconfiguration.json` and `awsconfiguration.json` to the Xcode window**, under your project's folder as seen in this screenshot:
+
+![GSA](~/images/project-setup/50_1_dragDrop.png)
+
+* Enable **Copy items if needed** if not already enabled
+* For “Added folders”, have **Create groups** selected.
+* For “Add to targets”, make sure the app target (**MyAmplifyApp**) is checked.
+
+Click **Finish** to add these files to your project as shown in this screenshot:
+
+![GSA](~/images/project-setup/50_2_addFiles.png)
+
+Now when you build (CMD+ b) and run (CMD + r) your application, you should not see any errors on the console.
```

Comment:
In other places you put the keystrokes (e.g. CMD + b) in backticks. Here your spacing is inconsistent. I did **control-r**. We should pick an approach and stick with it, site-wide.

Reply:
agree for consistency, don't have any opinion here.

Reply:
Also not going to update.